radarlabs/radar-sdk-ios
# App Attest

Changes needed:
- Only call `attestKey` once
- Call `generateAssertion` and pass that assertion to `POST /track` instead
- See https://developer.apple.com/documentation/devicecheck/preparing_to_use_the_app_attest_service?language=objc
